tests/conftest.py

```diff
@@ -110,6 +111,29 @@ def clients(django_db_setup, django_db_blocker, groups, token):
         User.objects.all().delete()
 
 
+@pytest.yield_fixture(scope="module", autouse=False)
+def permission_client(django_db_setup, django_db_blocker, token):
```

Comment:
Yes, I meant to make a comment about it. I would like to move everything to use this fixture instead of the `clients` fixture. Doing this means we can avoid the paramatrized tests that test administrators, investigators, developers, etc. that essentially duplicate the same tests and lock us into that set of permission groups. Doing things this way means we only need a test for has permission case and the does not have permission case plus it means that we may add, remove, and change permission groups as needed without updating tests.